lib/github.js

```diff
@@ -22,18 +36,20 @@ module.exports = {
     return !!commit;
   },
 
-  createStatus: async (owner, repo, sha, target_url) => {
-    let github = new Octokit({
-      auth: GITHUB_TOKEN
-    });
-    return github.repos.createCommitStatus({
+  createStatus: async (owner, repo, head_sha, entries) => {
+    const github = await getRestClient();
+    return github.rest.checks.create({
       owner,
       repo,
-      sha,
-      state: 'success',
-      target_url,
-      description: 'Deployed to Incrementals.',
-      context: 'continuous-integration/jenkins/incrementals'
+      name: 'Incrementals',
+      head_sha,
+      status: 'completed',
+      conclusion: 'success',
+      details_url: entries[0].url,
+      output: {
+        title: 'Deployed to Incrementals',
```

Comment:
I wonder if we should pick text that is more self-descriptive for people who do not already know what this means? Somehow indicating that this snapshot-ish build is available for download or consumption from Maven in draft PRs, linking to jenkins.io docs, etc.